addon/components/pix-stars.js

```diff
@@ -0,0 +1,24 @@
+import Component from '@glimmer/component';
+
+export default class PixStars extends Component {
+  get pixStarsClass() {
+    if (!this.args.color) return "pix-stars";
+    return `pix-stars pix-stars--${this.args.color}`;
+  }
+
+  get stars() {
+    const { count, total } = this.args;
+    
+    const starsCount = total || count;
+    
+    const stars = [];
+    for (let index = 0; index < starsCount; index++) {
+      if (index < this.args.count) {
+        stars[index] = 1
+      } else {
+        stars[index] = 0
+      }
+    }
+    return stars
```

Comment:
Je propose quelque chose de plus clair que 0 ou un pour déterminer si l'étoile est acquise ou non :
```suggestion
    for (let index = 0; index < starsCount; index++) {
      if (index < this.args.count) {
        stars[index] = 'acquired';
      } else {
        stars[index] = 'unacquired';
      }
    }
    return stars;
```

Reply:
En fait on a mis en place ce mécanisme pour gérer les futurs cas avec des quarts, demi ou 3/4 d'étoiles. Voire même pourquoi pas gérer un pourcentage d'avancement dans l'étoile.
Mais pour le moment, je peux prendre ta proposition qui permet une écriture plus facile dans le `hbs`

Reply:
Tu en penses quoi ?

Reply:
yes je pense qu'on pourra modifier le comportement en temps voulu :)